python/servo/platform/macos.py

```diff
@@ -52,7 +52,7 @@ def _platform_bootstrap(self, _force: bool) -> bool:
         installed_something |= self._platform_bootstrap_gstreamer(target, False)
         return installed_something
 
-    def _platform_bootstrap_gstreamer(self, target: BuildTarget, force: bool) -> bool:
+    def _platform_bootstrap_gstreamer(self, target: BuildTarget, force: bool) -> bool:  # pyrefly: ignore
```

Comment:
I don't think we should disable pyrefly's `bad-override` rule. It is still useful to catch issues where the base and derived classes have different signatures for overriden methods. My suggestion [here](https://github.com/servo/servo/pull/37729#discussion_r2192268954) was to just fix the *names* of the parameters so that the base class doesn't have the underscore and if that leads to the *linter* complain about unused variable, supress the liner for that line (not the typechecker) .